TM1py/Services/ServerService.py

```diff
@@ -184,6 +200,46 @@ def get_transaction_log_entries(self, reverse: bool = True, user: str = None, cu
             url += '&$top={}'.format(top)
         response = self._rest.GET(url, **kwargs)
         return response.json()['value']
+    
+    @require_admin
+    def get_audit_log_entries(self, user: str = None, object_type: str = None, object_name: str = None,
```

Comment:
Please add the `require_version `decorator as well.
PA 2.0.8 should map to  11.6
https://www.ibm.com/support/knowledgecenter/SSD29G_2.0.0/com.ibm.swg.ba.cognos.tm1_nfg.2.0.0.doc/c_pa_nfg_introduction.html

Reply:
Done